1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -501,3 +501,29 @@ Indicates whether to enable the DC/OS proxy.
 For more information, see the [examples](/docs/1.10/installing/custom/configuration/examples/#http-proxy).
 
 **Important:** You should also configure an HTTP proxy for [Docker](https://docs.docker.com/engine/admin/systemd/#/http-proxy).
+
+### <a id="zk-superuser"></a> zk_super_credentials (Enterprise DC/OS Only)
+
+In `strict` and `permissive` clusters the information stored in ZooKeeper is protected by ACLs so that a malicious user can't connect to ZooKeeper quorum and directly modify service metadata. ACLs specify sets of IDs and permissions that are associated with those IDs. ZooKeeper supports pluggable authentication schemes and has a few built in schemes: `world`, `auth`, `digest`, `host`, `ip`. 
+
+DC/OS ZooKeeper credentials `zk_super_credentials`, `zk_master_credentials`, and `zk_agent_credentials` use `digest` authentication, which require a `<uid>:<password>` string which is then used as an ID while checking if a client can access a particular resource.
```

Comment:
> use `digest` authentication, which require a [...] string

This isn't grammatically correct. If the meaning is that the authentication itself requires a string, change "require" to "requires".